multimedia/gpac/Portfile

```diff
@@ -18,12 +18,12 @@ long_description    GPAC is an Open Source multimedia framework for \
                     animation and interactivity). Includes MP4Box and \
                     Osmo4.
 
-homepage            http://gpac.wp.mines-telecom.fr/
+homepage            https://gpac.io
```

Comment:
This site doesn't seem to load... looks like https://gpac.wp.imt.fr/ instead

Reply:
Ah okay. Looks like the HTTPS version of that doesn't work. I've updated the PR. Thanks!